Lib/abc.py

```diff
@@ -28,7 +28,13 @@ def my_abstract_method(self, ...):
 class abstractclassmethod(classmethod):
     """A decorator indicating abstract classmethods.
 
-    Deprecated, use 'classmethod' with 'abstractmethod' instead.
+    Deprecated, instead use `classmethod` on top of `abstractmethod`:
```

Comment:
I'd rather we keep the original wording here, with the example.

(same for the other two doc-strings)